Jenkinsfile_nightly

```diff
@@ -93,13 +94,18 @@ withNightlyPipeline(type, product, component) {
         // hmcts/cnp-jenkins-library may fail to copy artifacts after checkstyle error so repeat command (see /src/uk/gov/hmcts/contino/GradleBuilder.groovy)
         steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/reports/checkstyle/*.html'
     }
-
-    after('functionalTest:preview') {
+    
+    enableSecurityScan()
+    
+    enableApiGatewayTest()
+    
+    enableMutationTest()
+    
+    enableFullFunctionalTest(200)
+    
+    after('fullFunctionalTest') {
         steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/target/cucumber/**/*'
```

Comment:
This folder may not be correct, as we have separate report folders for functional and smoke tests. pls see jenkins cnp file.